Assets/Scripts/Game/Items/ItemHelper.cs

```diff
@@ -974,6 +974,11 @@ public void AssignStartingGear(PlayerEntity playerEntity)
             equipTable.EquipItem(shortShirt, true, false);
             equipTable.EquipItem(casualPants, true, false);
 
+            // Add class-specific starting weapon
+            byte[] StartingWeaponTypesByClass      = { 3, 6, 6, 3, 11, 3, 3, 1, 6, 3, 3, 7, 2, 254, 14, 13, 5, 7, 7 };
+            byte[] StartingWeaponMaterialsByClass  = { 1, 0, 0, 1,  0, 0, 0, 1, 0, 0, 0, 0, 1, 254,  1,  0, 0, 0, 0 };
+            items.AddItem(ItemBuilder.CreateWeapon((Weapons)StartingWeaponTypesByClass[classIndex], (WeaponMaterialTypes)StartingWeaponMaterialsByClass[classIndex]));
```

Comment:
You need to handle the special case of the Archer, I think passing to AddItem will just cause an exception or something. You could just ignore the 254 value here and do a special case for the archer class.

Reply:
My mistake, I forgot the whole extra items for the archer part. In fact I had to redo that section so the proper values get passed.